components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-service-provider.jsp

```diff
@@ -1263,7 +1318,27 @@
                                 </div>
                             </td>
                         </tr>
+
+                        <!-----Add radio button to select certificate or jwks end point fro sp--------------->
```

Comment:
Shall we change this comment to the following?
`<!-- Add a radio button to select the certificate or JWKS endpoint from sp -->`

Reply:
done